lib/HistoryModelService.js

```diff
@@ -5,90 +5,53 @@ var HistoryModelFields = require('./HistoryModelFields');
 var WaterlineLifecycleCallback = require('./WaterlineLifecycleCallback');
 
 var self = {
-  createModels: function createModels(sails) {
+  createModels: function createModels(sails, callback){
     async.auto({
-      loadCollection: createAllModelHistory,
-      loadModel: ['loadCollection', loadModels]
+      loadCollection: function createAllModelHistory(callback){
```

Comment:
`callback` variable is already declared in the parent function. Please rename the variable.